sdk/storage/storage-file-share/test/fileclient.spec.ts

```diff
@@ -294,6 +294,34 @@ describe("FileClient", () => {
     assert.deepStrictEqual(properties2.copySource, fileClient.url);
   });
 
+  it("startCopyFromURL with smb options", async () => {
+    await fileClient.create(1024);
+    const newFileClient = dirClient.getFileClient(recorder.getUniqueName("copiedfile"));
+
+    const fileAttributes = "Hidden | System";
```

Comment:
Other languages have `NTFSAttributes` to help build this fileAttributes string.  Do we have similar?
https://github.com/Azure/azure-sdk-for-java/pull/7589/files#diff-0309bd374a72157c1c4c5a7f9b563eb3R341

Reply:
.NET used a bitmap for this. Could we have a neat solution in JavaScript?

Reply:
Good news, we have `FileSystemAttributes`.